x-pack/plugins/security_solution/public/entity_analytics/components/severity/common/index.test.tsx

```diff
@@ -13,10 +13,9 @@ import { TestProviders } from '../../../../common/mock';
 import type { EuiHealthProps } from '@elastic/eui';
 import { EuiHealth } from '@elastic/eui';
 
-import { euiThemeVars } from '@kbn/ui-theme';
 import { RiskSeverity } from '../../../../../common/search_strategy';
 import { RiskScoreLevel } from '.';
-import { SEVERITY_COLOR } from '../../../../overview/components/detection_response/utils';
+import { RISK_SEVERITY_COLOUR } from '../../../common/utils';
```

Comment:
Changes from SEVERITY_COLOR to this as the constant was changed to this 11 months ago and not updated in test, now consistent and removes use of dependance on euiThemeVars. Updated all consts along with it.